test/e2e/util.go

```diff
@@ -52,11 +52,11 @@ var (
 	pollLongTimeout = 5 * time.Minute
 )
 
-// type ConditionFunc func() (done bool, err error)
-// Poll(interval, timeout time.Duration, condition ConditionFunc)
+// type wait.ConditionWithContextFunc func(context.Context) (done bool, err error)
+// PollUntilContextTimeout(ctx context.Context, interval, timeout time.Duration, immediate bool, condition wait.ConditionWithContextFunc)
```

Comment:
These two comment lines seem a little misplaced. Can we just remove them?

Reply:
Sure

Reply:
Done